src/backend/distributed/sql/citus--11.0-2--11.1-1.sql

```diff
@@ -8,3 +8,5 @@ DROP FUNCTION pg_catalog.worker_range_partition_table(bigint, integer, text, tex
 DROP FUNCTION pg_catalog.worker_repartition_cleanup(bigint);
 
 #include "../../columnar/sql/columnar--11.0-2--11.1-1.sql"
+#include "udfs/citus_split_shard_by_split_points/11.0-2.sql"
+#include "udfs/worker_split_copy/11.0-2.sql"
```

Comment:
I think we're missing the downgrade script

Reply:
@marcocitus , Any existing example I can look into for downgrade?

Reply:
sql/downgrades has all the scripts. You mainly need to drop the function to revert the change.

Reply:
So a function introduced in citus--11.0.2--11.1.1 will go to corresponding downgrades/ file  ?

Reply:
@marcocitus, So a function introduced in citus--11.0.2--11.1.1 will go to corresponding downgrades/ file ?

Reply:
Done.